### DISCUSSION TITLE
v6：errors更新机制

### DISCUSSION THREAD

**Discussion by amberlwan** [react-hook-form/react-hook-form#6314] [General]
v6：想了解下errors为啥会是异步更新的呢，在react项目下搜集errors的变更只能通过监听formState的变更去更新？

**Comment by bluebill1049** [comment]
because we are batch update formState, this can be improved after react 18.

**Comment by amberlwan** [comment]
另外请教下，formState里面的状态都是异步更新的吗？


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "react-hook-form/react-hook-form"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年8月23日(星期一) 晚上9:38
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [react-hook-form/react-hook-form] v6：errors更新机制 (#6314)





 
because we are batch update formState, this can be improved after react 18.
 
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
